README.md

```diff
@@ -111,7 +111,7 @@ For a minimalistic working demo, check the
 - `gpu`: [Optional] A template string that will be used to request GPU resources
   through `--gres`. The template should therefore include a `{}` that will be
   replaced by the number of requested GPU **and** follow the format expected by
-  `--gres`. If no GPU is available for this partition, set to `None`. It is
+  `--gres`. If no GPU is available for this partition, set to `""`. It is
```

Comment:
There is a behavior change here: `None` means probe from SLURM... `""` means disable GPUs.
To me this is affordable.

Reply:
You will have to explain this to me but I trust you :+1: